network/src/peer_manager/builder.rs

```diff
@@ -232,14 +196,13 @@ impl PeerManagerBuilder {
         Self {
             network_context,
             time_service,
-            transport_context: Some(TransportContext::new(
```

Comment:
Don't like the `new` pattern?

Reply:
at least here it was just extra random boilerplate imo